manala.skeleton/CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+## [1.0.34] - 2019-09-18
+### Fixed
+- Update dependency roles names
```

Comment:
A bite late, sorry, but it looks like a weird/copy/paste changelog entry :)

Reply:
yes indeed, I'll make a PR to fix it